.github/workflows/visual-audit.yml

```
@@ -0,0 +1,72 @@
+          "output=$output" >> $env:GITHUB_OUTPUT
+
+      - name: Compare visual pack
+        shell: pwsh
+        run: |
+          pwsh ./tools/compare-visual-pack.ps1 `
+            -BaselineDir "${{ steps.paths.outputs.baseline }}" `
+            -CandidateDir "${{ steps.paths.outputs.candidate }}" `
+            -OutputPath "${{ steps.paths.outputs.output }}"
+
+      - name: Applitools availability note
+        if: ${{ secrets.APPLITOOLS_API_KEY != '' }}
```

actionlint (1.7.10)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)